src/components/NavBar/NavBar.jsx

```diff
@@ -1,14 +1,23 @@
 import './NavBar.styles.css'
 
-export const NavBar = ({ setShowRules }) => {
+export const NavBar = ({ setShowRules, setDarkMode, darkMode }) => {
     return (
         <nav className="navbar">
+            <h1 className='title'>Weldor</h1>
             <ul>
```

Comment:
Chrome: renders to the right of the centered title making the UI unbalanced.
![image](https://github.com/user-attachments/assets/1fe5aa4e-3756-4d17-b859-1e4c950243bc)
Firefox: r/softwaregore.
![image](https://github.com/user-attachments/assets/d8563f1d-6ecf-44b3-a9a7-622737c19a34)
Consider moving the buttons to their own row below the title.

Reply:
Resolved by changing layout, thank you.